proxy/http/HttpSM.cc

```diff
@@ -7025,6 +7032,9 @@ HttpSM::setup_blind_tunnel(bool send_response_hdr, IOBufferReader *initial)
   c_os = tunnel.add_consumer(server_entry->vc, ua_entry->vc, &HttpSM::tunnel_handler_ssl_consumer, HT_HTTP_SERVER,
                              "http server - tunnel");
 
+  ua_entry->vc->make_tunnel_endpoint();
+  server_entry->vc->make_tunnel_endpoint();
```

Comment:
server_entry->vc is a VConnection pointer, which is why make_tunnel_endpoint() has to be virtual member function of VConnection.